kolla/defaults.yml

```diff
@@ -345,6 +346,10 @@ enable_k3s: no
 enable_zun: no
 enable_zun_compute_k8s: "{{ enable_zun | bool and enable_k3s | bool }}"
 blazar_enable_device_plugin_k8s: "{{ enable_k3s | bool }}"
+# K8S worker tainting
+zun_enforce_worker_taint: false
+doni_enable_worker_taint: false
+worker_taint: "{{ k8s_worker_taint.key | default('worker-node') }}={{ k8s_worker_taint.value | default('true') }}:{{ k8s_worker_taint.effect | default('NoSchedule') }}"
```

Comment:
Is this a string or a dict? you reference `worker_taint.key` in doni.conf?

Reply:
this is a dict, looks like my merge was a little destructive